tensorflow-framework/src/test/java/org/tensorflow/framework/activations/ReLUTest.java

```diff
@@ -82,7 +82,7 @@ public void testCall__Long() {
 
   /** Test of ReLU call method */
   @Test
-  public void testCall__Float16() {
+  public void testCall_Float16() {
```

Comment:
Still snake_case, looks like the find replace didn't get all of them.

Reply:
Fixed